docs/user/dashboard/lens.asciidoc

```diff
@@ -22,6 +22,51 @@ image::images/lens_data_info.png[Data summary window with Other]
 
 NOTE: The sum of all the *Other* fields can equal more than 100% by a small amount.
 
+[float]
+[[drag-and-drop-keyboard-navigation]]
+=== Create your visualizations faster with keyboard navigation
+
+Lens has a fully accessible, continuously improved drag and drop system. By using keyboard navigation instead of a mouse you can significantly speed up your work.
+
+[role="screenshot"]
+image::images/lens_drag_drop_1.png[Presented Lens drag and drop]
+
+[float]
+[[drag-and-drop-keyboard-navigation-drop-a-field]]
+==== Drop a field to the workspace or configuration panel
+
+. Set a focus on the chosen field. 
+. Press space to start a drag.
+. Using left and right arrows, select a drop target. 
+. Press space to confirm.
+
+[float]
+[[drag-and-drop-keyboard-navigation-duplicate-an-operation]]
+==== Duplicate an existing operation
+
+Creating a few similar operations can be repetetive and time-consuming. A good usecase would be a need for creating multiple percentiles. Instead of configuring each metric from scratch, you can duplicate it – just drag a metric to duplicate and drop it into an empty dimension. For keyboard navigation:
```

Comment:
```suggestion
Creating a few similar operations can be repetitive and time-consuming. A good use case would be a need for creating multiple percentiles. Instead of configuring each metric from scratch, you can duplicate it – just drag a metric into an empty dimension to duplicate it. For keyboard navigation:
```